### DISCUSSION TITLE
Question regarding LaTeX

### DISCUSSION THREAD

**Discussion by Charango-Chile** [quarto-dev/quarto-web#129] [Q&A]
Hi everyone,

I guess that when using $$ $$ to enclose equations in revealjs then what we should get is a centered equation. For some reason I'm not quite getting that at the moment and the equations are left justified.

Is there any global option that ensures that equations will be centered when using diplay mode?

Cheers, Jose

**Comment by Charango-Chile** [comment]
Hi,

After trial and error, by undoing things I added to a personalised scss file, I discovered that if I set for the element `.reveal p {` the option `display: inline-block;` then the display equations are left-aligned. The problem went away when I removed the display setting.

Cheers, Jose

**Reply by jjallaire** [reply]
Great, glad this works as expected now!

**Reply by Charango-Chile** [reply]
Thanks for answering us... You are a champ!

Cheers, Jose
